### ISSUE TITLE
comma at wrong posision

### ISSUE THREAD

**Issue by droon5** [Exa-Networks/exabgp#396]
Hello,

I have two exaBGP machines, the first is sending flow route updates to the second one. The second runs python program which receives parsed updates in JSON format.

If the first send this route update (from /qa/conf/flow.conf):
```
flow {
		route first-vpn-route {
			rd 65535:65536;
			match {
				source 10.0.0.1/32;
			}
			then {
				discard;
			}
		}
	}
```
the second will parse it like this:
```
{ "exabgp": "3.4.8", "time": 1459241686, "host" : "tiger", "pid" : "9601", "ppid" : "9172", "counter": 10, "type": "update", "header": "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF004702", "body": "000000304001010040020040050400000064C010088006000000000000800E1400018600000E0000FFFF0001000002200A000001", "neighbor": { "ip": "192.168.51.94", "address": { "local": "192.168.51.96", "peer": "192.168.51.94"}, "asn": { "local": "65000", "peer": "65000"}, "message": { "update": { "attribute": { "origin": "igp", "local-preference": 100, "extended-community": [ 9225060886715039744 ] }, "announce": { "ipv4 flow-vpn": { "none": { "flow-3": {, "route-distinguisher": "65535:65536" "source": [ "10.0.0.1/32" ], "string": "flow route-distinguisher 65535:65536 source 10.0.0.1/32" } } } } } }} }
```

There is comma at the begin on "flow-3" block. I suppose it should be between "route-distinguisher" and "source".

Also there is wrong exaBGP version, I use release 3.4.16.

Thanks

**Comment by thomas-mangin** [comment]
The version is right (if confusing) it is the latest exabgp version which modified the wire format. So you know that if the number does not change you code will run :-)

For the missing / wrong comma - looking into it.

**Comment by droon5** [comment]
Ok, thanks :-)

**Comment by thomas-mangin** [comment]
Thank you - this is now fixed both on the 3.4 and master.
